model/model.go

```diff
@@ -287,6 +287,7 @@ type PartitionDefinition struct {
 	ID       int64    `json:"id"`
 	Name     string   `json:"name"`
 	LessThan []string `json:"less_than"`
+	MaxValue bool     `json:"max_value"`
```

Comment:
s/MaxValue/IsMaxValue/

Reply:
What a terrible idea to have this field in `PartitionDefinition`.
It's redundancy information, it's noise.